modules/node_modules/@ncigdc/uikit/withToggle.js

```diff
@@ -1,20 +1,42 @@
 import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
 
 export default function withToggle(ComposedComponent) {
   return class Toggle extends Component {
+    static propTypes = {
+      // Allow active state changes to trigger actions such as toggling relay variables
+      onActivate: PropTypes.func,
+      onDeactivate: PropTypes.func,
+      active: PropTypes.bool,
+    };
+
+    static defaultProps = {
+      onActivate: _.noop,
```

Comment:
I don't have strong preferences on this one. Academic memory and initialization benefits of sharing a single instance of noop instead of creating separate functions, but that problably comes down to less than a few kb and a few ms